### ISSUE TITLE
Cannot launch RStudio Daily; relocation error: Qt_5_PRIVATE_API not defined in file libQt5Core.so.5 with link time reference

### ISSUE THREAD

**Issue by chriselrod** [rstudio/rstudio#2088]
### System details

    RStudio Edition : Desktop
    RStudio Version :  Daily: 1.2.305
    OS Version      :  Fedora 27
    R Version       :  3.4.3

### Steps to reproduce the problem
From the command line:
```
$ rstudio
rstudio: relocation error: /lib64/libQt5XmlPatterns.so.5: symbol _ZN9QXmlUtils8isNCNameERK10QStringRef, version Qt_5_PRIVATE_API not defined in file libQt5Core.so.5 with link time reference
```
If trying to open RStudio from the activities pane, nothing happens.

### Describe the problem in detail

I installed the latest release version of RStudio, but it crashed every five or so minutes.
That was annoying, so I figured I'd try the daily. Dailies aren't known for their stability, but before complaining too loudly about a bug  I figured I should see whether it's been fixed in the development version. (Of course, the problem may be in one of the dependencies, rather than RStudio itself.)

I could not open the daily, so I figured I'd try from the command line:
```
$ rstudio
rstudio: error while loading shared libraries: libQt5WebEngine.so.5: cannot open shared object file: No such file or directory
$ sudo dnf qt5-qtwebengine-devel
$ rstudio
rstudio: error while loading shared libraries: libQt5Sensors.so.5: cannot open shared object file: No such file or directory
$ sudo dnf qt5-devel
$ rstudio
rstudio: relocation error: /lib64/libQt5XmlPatterns.so.5: symbol _ZN9QXmlUtils8isNCNameERK10QStringRef, version Qt_5_PRIVATE_API not defined in file libQt5Core.so.5 with link time reference
```

### Describe the behavior you expected

I expect RStudio to launch and not crash.
The crashing did not coincide with R doing any actual work, but sporadically.
Eg, often when I was looking at a browser or pdf document.

I could switch back to the release version, but figured I'd focus the issue on the Qt-5 issue, as it seems more severe.

**Comment by kevinushey** [comment]
It looks like our Fedora binaries were not bundling a few requisite Qt libraries. I just made a couple fixes that should hopefully ensure the next daily builds on Fedora are in good working order.

**Comment by kevinushey** [comment]
Should be fixed up with the latest daily builds (v1.2.315). If you're still seeing this issue can you please re-open?

**Comment by chriselrod** [comment]
Launches. Seems to run fine. No crashes (but that'd be a separate issue). Thanks!
